feat(web-server): Allow Range headers in web server.

TheModMaker opened the pull request:
Adds the ability to use Range headers in the web server (#2140).

dignifiedquire commented:
Thanks, could you also add a note to the documentation so users know this exists and works please?

TheModMaker commented:
I couldn't find an obvious place to add to the documentation, so I added a section in the Files file.

Also, re-reading the spec, my handling of multiple ranges was incorrect.  So I removed support for multiple ranges since it would be more complicated to add it.

dignifiedquire commented:
Thanks :octocat:
